src/main/java/org/ehcache/sizeof/ObjectGraphWalker.java

```diff
@@ -263,6 +263,14 @@ private static void nullSafeAdd(final Deque<Object> toVisit, final Object o) {
                         LOG.error("Security settings prevent Ehcache from accessing the subgraph beneath '{}'" +
                                   " - cache sizes may be underestimated as a result", field, e);
                         continue;
+                    } catch (RuntimeException e) {
+                        // new class (jdk9+ only) can be thrown
+                        if ("InaccessibleObjectException".equals(e.getClass().getSimpleName())) {
+                            LOG.error("Field access cannot be enabled. This prevents Ehcache from accessing "
```

Comment:
Do we want to be explicit about referencing the JPMS in this message rather than just referring to "Field access"?

Reply:
It is up to the project policy, I say more info does not hurt. Imho, the clearest message is the original exception's message.

Reply:
```suggestion
                            LOG.error("JPMS blocks field access. This prevents Ehcache from accessing "
```